doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -105,6 +105,7 @@ Matter
 * Disabled OpenThread shell by default in Matter over Thread samples.
 * Enabled :kconfig:option:`CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option by default, including for builds without factory data support.
   The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
+* Fixed an application core crash on nRF5340 targets with the factory data module enabled, after the OTA firmware update finishes and the image is confirmed.
```

Comment:
Both Fixed entries should be combined under the same bullet. This removes Line 104.
```suggestion
* Fixed:
  * An IPC crash on nRF5340 when Zephyr's main thread takes a long time.
  * An application core crash on nRF5340 targets with the factory data module enabled.
    The crash would happen after the OTA firmware update finishes and the image is confirmed.
```